server/routes/shared/action-plan/actionPlanPresenter.ts

```diff
@@ -14,7 +16,8 @@ export default class ActionPlanPresenter {
     private readonly actionPlan: ActionPlan,
     private readonly serviceCategories: ServiceCategory[],
     readonly userType: 'service-provider' | 'probation-practitioner',
-    private readonly validationError: FormValidationError | null = null
+    private readonly validationError: FormValidationError | null = null,
+    private readonly approvedActionPlanSummaries: ApprovedActionPlanSummary[] | null = null
```

Comment:
Rather than null it would be easier to defaulting to empty list so you don't need to handle null value. (that's what you pass from probationPractitionerReferralsController.ts if feature is off)

Reply:
I went back and forth on this, started out with an empty list and changed it to `null` - I agree, I think it is easier and we have fewer `null` checks. I changed it because I thought it might be useful to be able to distinguish between having no  previously-approved action plan summaries or it being `null` because it's an SP viewing the page and shouldn't see this section. I think I'm going to change it back though as I don't think the distinction is that useful, particularly as we can use the `userType`.

Reply:
ah I see your point -  I was thinking the null was there because of the feature flag; so in the way you describe a null does make sense. If you want to keep it as a null then I'm happy with that too.

Reply:
Have changed this back to an empty array